mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/services/ArenaAdapterService.kt

```diff
@@ -47,13 +52,22 @@ class ArenaAdapterService(
         return avtaler.delete(id)
     }
 
-    fun upsertTiltaksgjennomforing(tiltaksgjennomforing: TiltaksgjennomforingDbo): QueryResult<TiltaksgjennomforingAdminDto> {
+    suspend fun upsertTiltaksgjennomforing(tiltaksgjennomforing: TiltaksgjennomforingDbo): QueryResult<TiltaksgjennomforingAdminDto> {
         return tiltaksgjennomforinger.upsert(tiltaksgjennomforing)
             .flatMap { tiltaksgjennomforinger.get(tiltaksgjennomforing.id) }
             .map { it!! }
             .onRight {
                 tiltaksgjennomforingKafkaProducer.publish(TiltaksgjennomforingDto.from(it))
             }
+            .onRight {
+                if (it.sluttDato == null || it.sluttDato?.isAfter(LocalDate.of(2023, 1, 1)) == true) {
```

Comment:
Kunne vi trukket ut denne `LocalDate.of(2023, 1, 1)` til en `class/object` slik at vi får dokumentert denne "vilkårlige" datoen litt bedre? Denne er i sync med `AdminTiltaksgjennomforingFilter.sluttDatoCutoff` som sikkert også burde referere til samme konstant.

Er usikker på navngiving, men kanskje en egen fil med litt arenamigreringsspesifikk kode/dokumentasjon ? Feks
```
object Arenamigrering {
    val TiltaksgjennomforingSluttDatoCutoff = LocalDate.of(2023, 1, 1)
}
```

Reply:
Høres lurt ut